test/src/userobjects/RestartableTypes.C

```diff
@@ -79,6 +80,10 @@ void RestartableTypes::initialSetup()
   _map_data[1] = 2.2;
   _map_data[2] = 3.4;
 
+  _dense_vector_data.resize(3);
+  for (unsigned int i = 0; i < _dense_vector_data.size(); i++)
+    _dense_vector_data(i) = (Real) (i);
```

Comment:
This isn't really testing that it's restartable.

You also need to increment the values in `timestepSetup()` and then check for the incremented values in `RestartableTypesChecker`.